scripts/setup-ubuntu.sh

```diff
@@ -65,12 +65,9 @@ PACKAGES+=" xmlto"
 PACKAGES+=" xmltoman"
 
 # Needed by python modules (e.g. asciinema) and some build systems.
-PACKAGES+=" python3.10"
-PACKAGES+=" python3.11"
```

Comment:
This line should not be deleted. `python3.11-venv` depends on `python3.11` so it will actually be installed, but it is unclear for us.

Reply:
👍 - added it back in 7b926a35419d4b2c475a45a6e3cab985370854c0.